from functions import test_me


assert not test_me(5)
assert test_me(10)


# raise Exception("Hello there")